One table

| no. | error |
|-:|:-|
| 63 | AttributeError: module 'torch.jit._state' has no attribute '_clear_class_state' |
| 38 | RuntimeError: iter.device(arg).is_cuda() INTERNAL ASSERT FAILED at "/pytorch/aten/src/ATen/native/cu |
| 3 | OSError: gs555750 is not a valid git identifier (branch name, tag name or commit id) that exists for |
| 3 | AssertionError: Couldn't trace module. |
| 3 | RuntimeError: "normal_kernel_cpu" not implemented for 'BFloat16' |
| 1 | RuntimeError: Caught RuntimeError in replica 0 on device 0. |

--- a/utils/get_ci_error_statistics.py
+++ b/utils/get_ci_error_statistics.py
@@ -26,7 +26,7 @@ def get_artifacts_links(worflow_run_id):
             artifacts.update({artifact["name"]: artifact["archive_download_url"] for artifact in result["artifacts"]})
 
         return artifacts
-    except Exception as e:
+    except Exception:
         print("Unknown error, could not fetch links.", e)
 
     return {}
@@ -175,7 +175,7 @@ def make_github_table_per_model(reduced_by_model):
         count = reduced_by_model[model]["count"]
         try:
             error, _count = list(reduced_by_model[model]["errors"].items())[0]
-        except Exception as e:
+        except Exception:
             pass
         line = f"| {model} | {count} | {error[:60]} | {_count} |"
         lines.append(line)

--- a/utils/get_ci_error_statistics.py
+++ b/utils/get_ci_error_statistics.py
@@ -26,7 +26,7 @@ def get_artifacts_links(worflow_run_id):
             artifacts.update({artifact["name"]: artifact["archive_download_url"] for artifact in result["artifacts"]})
 
         return artifacts
-    except Exception:
+    except Exception as e:
         print("Unknown error, could not fetch links.", e)
 
     return {}

--- a/utils/get_ci_error_statistics.py
+++ b/utils/get_ci_error_statistics.py
@@ -156,7 +156,7 @@ def reduce_by_model(logs, error_filter=None):
 
 
 def make_github_table(reduced_by_error):
-    header = f"| no. | error |"
+    header = "| no. | error |"
     sep = "|-:|:-|"
     lines = [header, sep]
     for error in reduced_by_error:
@@ -168,14 +168,14 @@ def make_github_table(reduced_by_error):
 
 
 def make_github_table_per_model(reduced_by_model):
-    header = f"| model | no. of errors | major error | count |"
+    header = "| model | no. of errors | major error | count |"
     sep = "|-:|-:|-:|-:|"
     lines = [header, sep]
     for model in reduced_by_model:
         count = reduced_by_model[model]["count"]
         try:
             error, _count = list(reduced_by_model[model]["errors"].items())[0]
-        except:
+        except Exception as e:
             pass
         line = f"| {model} | {count} | {error[:60]} | {_count} |"
         lines.append(line)

